tests/ui/consts/large_const_alloc.rs

```diff
@@ -2,6 +2,7 @@
 // on 32bit and 16bit platforms it is plausible that the maximum allocation size will succeed
 // FIXME (#135952) In some cases on AArch64 Linux the diagnostic does not trigger
 //@ ignore-aarch64-unknown-linux-gnu
+//@ ignore-aix: FIXME(#137966)
```

Comment:
There is nothing to fix here.
```suggestion
//@ ignore-aix: alloc failure on AIX can result in SIGKILL instead of nullptr
```